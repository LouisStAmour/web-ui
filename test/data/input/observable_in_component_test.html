<!doctype html>
<!--
   Copyright (c) 2012, the Dart project authors.  Please see the AUTHORS file
   for details. All rights reserved. Use of this source code is governed by a
   BSD-style license that can be found in the LICENSE file.
-->
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <script type="application/javascript" src="testing.js"></script>
</head>
<body>
  <element name="x-foo" constructor="Foo">
    <template>Test status: {{status}}</template>
    <script type="application/dart" src="observable_in_component.dart"></script>
  </element>
  <x-foo></x-foo>
  <script type="application/dart">
    import 'dart:async';
    import 'dart:html';
    import 'package:unittest/unittest.dart';
    import 'observable_in_component.dart';

    main() {
      Timer.run(() {
        expect(status, 'FAIL');
        status = 'PASS';
        window.postMessage('done', '*');
      });
    }
  </script>
</body>
</html>
