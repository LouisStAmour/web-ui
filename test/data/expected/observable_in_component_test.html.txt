Content-Type: text/plain
<html lang="en"><head><style>template { display: none; }</style>
  <meta charset="utf-8">
  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
  <script src="../input/packages/web_ui/testing/testing.js"></script>
</head>
<body>
  
  <span is="x-foo"><shadow-root>Test status: PASS</shadow-root></span>
  


<script type="text/javascript" src="packages/web_ui/shadowdom.debug.js"></script>
<script type="application/dart" src="observable_in_component_test.html_bootstrap.dart"></script></body></html>
